### PR TITLE
Updated testing docs

### PR DESCRIPTION
**Pull request by osundwajeff**
Updated testing section in the docs

### CODE REVIEW

File: docs/src/developer/guide/testing.md [hunk 1]
```diff
@@ -1,66 +1,92 @@
...
 
 Once `Node.js` is installed, use `npm` JavaScript package manager to install `playwright`.
 
+To install dependencies defined in `package.json` file:
+
+```bash
+npm install
+```
+
+To install playwright browsers and OS secific depndencies:
```
Comment (Jeremy-Prior) [inline docs/src/developer/guide/testing.md:39]:
```suggestion
To install playwright browsers and OS specific dependencies:
```

File: docs/src/developer/guide/testing.md [hunk 2]
```diff
@@ -1,66 +1,92 @@
...
 
 To generate test reports:
 
 ```bash
-$ npx playwright show-report
+npx playwright show-report
 ```
 
-### Continuous intergration github actions
+### Continuous intergration testing
```
Comment (Jeremy-Prior) [inline docs/src/developer/guide/testing.md:89]:
```suggestion
### Continuous integration testing
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 19 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 27 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 16 min

**Review by Jeremy-Prior — APPROVED** [review]
Docs LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 50 sec

**Pull request merged**